### ISSUE TITLE
SF_ACC - ui5-shellbar: Buttons outline on focus is not having color contrast ratio 4.5:1 

### ISSUE THREAD

**Issue by pankajpandey01** [SAP/ui5-webcomponents#5795]
### **Bug Description**
ui5-shellbar- Buttons outline on focus is not having color contrast ratio 4.5:1 

### **Expected Behavior**
color contrast ratio should be at least 4.5:1 

### **Steps to Reproduce**
1. Go to 'https://sap.github.io/ui5-webcomponents/playground/components/ShellBar/' (Theme: Quartz Light)
2. Navigate to buttons like 'Notification', Search ... with keyboard
3. Check color contrast ratio of button's focus outline 

### **Context**
- UI5 Web Components version: {1.7.0}

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: {SuccessFactors}
- Business impact: {Accessibility issue}

**Comment by LidiyaGeorgieva** [comment]
Hello @ui5-webcomponents-topic-p please take over.
The color of the focus is not correct.

**Comment by pankajpandey01** [comment]
Hello, Please try to complete by Sep.15 and downport it to 1.6.x , if possible, as this is higher urgency for SFSF. Thank you.
